code/game/objects/effects/alien/aliens.dm

```diff
@@ -427,9 +424,6 @@
 			visible_message("<span class='alium'>[src.target] begins to crumble under the acid!</span>")
 	spawn(rand(150, 200)) tick()
 
-<<<<<<< HEAD
-//Xenomorph Effect egg removed, replaced with Structure Egg.
```

Comment:
Eggs were removed entirely on vorestation side previously, I think the PR just brought them back because diff.

Reply:
not much point in deleting them then, if diffs are just gonna bring them back again

just don't use them in any of our own POIs